internal/indexmanagement/reconcile.go

```diff
@@ -336,13 +336,27 @@ func (imr *IndexManagementRequest) reconcileIndexManagementCronjob(policy apis.I
 		{Name: "POLICY_MAPPING", Value: mapping.Name},
 	}
 
-	if policy.Phases.Delete != nil {
+	if policy.Phases.Delete != nil { //#BEE: I think I need to read namespace value here
+		var namespaceNames []string
+		var minAgeValue string
+		for _, namespaces := range policy.Phases.Delete.Namespace { //#BEE: I dont know why policy.Phases.Delete.Namespace is an array
+			namespaceNames = namespaces.NamespacesToDelete //#BEE: this is already an array..
+			minAgeValue = string(namespaces.MinAge)
+		}
+
+		namespaceNamesJson, err := json.Marshal(namespaceNames)
+		if err != nil {
+			return err
+		}
+
 		minAgeMillis, err := calculateMillisForTimeUnit(policy.Phases.Delete.MinAge)
 		if err != nil {
 			return err
 		}
 		envvars = append(envvars,
 			corev1.EnvVar{Name: "MIN_AGE", Value: strconv.FormatUint(minAgeMillis, 10)},
+			corev1.EnvVar{Name: "NAMESPACE_NAME", Value: string(namespaceNamesJson)}, //#BEE: now I can use this in the delete script
```

Comment:
The env var name `NAMESPACE_NAME` does not represent the purpose of `namespaceNamesJson` well enough. Latter is a list of namespaces while former sounds like a single name. I suggest to rename to something in plural case: `NAMESPACE_SPECS`, `NAMEPACES`...